PointsOnAMapBlog/Pages/PointOnAMap.razor

```diff
@@ -0,0 +1,36 @@
+﻿@page "/"
+@using Point = dymaptic.GeoBlazor.Core.Components.Geometries.Point
+
+<PageTitle>Point on a Map</PageTitle>
+
+<h1>Put a single Point on the map</h1>
+
+
+<MapView @ref="_mapView" Longitude="0" Latitude="0" Zoom="3" Style="height: 600px; width: 1200px;">
+    <Map>
+        <Basemap>
+            <BasemapStyle Name="BasemapStyleName.ArcgisTopographic" />
+        </Basemap>
+    </Map>
+</MapView>
+
+@code {
+    protected override async Task OnAfterRenderAsync(bool firstRender)
```

Comment:
Just a heads up, if you tried to do any ArcGIS querying of any sort in `OnAfterRender` it might throw an exception. We also have `OnViewRendered` on `MapView` that can be used for events after the map is fully rendered.